frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/object-service-dashboards.spec.ts

```diff
@@ -0,0 +1,149 @@
+import { SECOND } from '../../integration-tests/utils/consts';
+
+const testName = `test-${Math.random()
+  .toString(36)
+  .replace(/[^a-z]+/g, '')
+  .substr(0, 5)}`;
+
+const testBucket = {
+  apiVersion: 'objectbucket.io/v1alpha1',
+  kind: 'ObjectBucketClaim',
+  metadata: {
+    namespace: 'openshift-storage',
+    name: testName,
+  },
+  spec: {
+    ssl: false,
+    storageClassName: 'openshift-storage.noobaa.io',
+    generateBucketName: 'test-bucket',
+  },
+};
+
+const installStorageCluster = () => {
+  cy.login();
+  cy.visit('/');
+  cy.install();
+};
+
+describe('Tests Buckets Card', () => {
+  before(() => {
+    installStorageCluster();
+  });
+
+  beforeEach(() => {
+    cy.visit('/');
+    cy.byLegacyTestID('horizontal-link-Object Service').click();
+    cy.wait(10 * SECOND);
```

Comment:
Confirmed this a couple of times and both `it` need a 10 second buffer time for the page to load.

Reply:
Wait for the elements that you want to test. For example if you want to test the details card then wait on the details card to load. Don't wait arbitrarily.

Reply:
Right, that's better. Sending a patch soon.